pythia/tasks/samplers.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
```

Comment:
This has landed in master in https://github.com/pytorch/pytorch/pull/22479
But, since we are not yet on nightly build, I will keep this for now.